### PR TITLE
Feature/local storage

### PR DESCRIPTION
**Pull request by atownse**
## Is this a fix or a feature?
Feature

## What is the change?
The logged activity cards stay displayed on page reload

## What does it fix? 
N/A

## Where should the reviewer start?
activity.js: line 21; main.js: line 199

## How should this be tested?
log an activity, then refresh the page

### Screenshot (optional)

@sarah01123
@elle624
@atownse

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -196,31 +196,36 @@ function startCountDown() {
   }
 }
 
+function returnFromLocalStorage() {
```
Comment (yiranli624) [inline main.js:199]: We will have to come back and refactor this

**Review by yiranli624 — APPROVED** [review]
We did great on figuring out the localStorage!

File: activity.js [hunk 2]
```diff
@@ -14,12 +14,13 @@ class Activity {
     }
 
     markComplete() {
-        if (this.seconds < 0) {
-            this.completed = true;
-        }
+        this.completed = true;
```
Comment (saraho1123) [inline activity.js:17]: We should come back to this and consider if we are updating the data model vs the DOM, and possibly refactor a bit.

**Review by saraho1123 — APPROVED** [review]
We did it team! it5 is working. I'm so excited we get to refactor and make it even better! :

File: main.js (hunk 1 above)
Comment (saraho1123) [reply main.js:199]: agreed!

**Pull request merged**